handbook/ce/ce_to_eng_handover.md

```diff
@@ -0,0 +1,35 @@
+# CE-to-Engineering issue handover process
+
+When a prospect or customer files a new support ticket or raises a new issue, CE and Engineering should follow the steps below to ensure the issue is handled successfully.
+
+## Principles
+
+**CE is always the first line of defense.**
+
+**Engineering should only feel a responsibility to get involved if tagged in by CE.**
+
+**However, once someone is "assigned" to a ticket (whether formally, or they informally take over the conversation), it is up to them to either (1) see the issue through to resolution or (2) assign a new owner.**
```

Comment:
Can we change the wording so it is clear that if someone "informally" takes over the conversation, then they should be formally assigned the ticket? I want to avoid grey areas on who is responsible.